src/pkt.c

```diff
@@ -181,6 +181,12 @@ void handle_packet(char *buf, unsigned bytes, int is_pcap, struct sockaddr_in *a
 							pt_log(kLog_error, "Failed to create proxy descriptor!\n");
 							return;
 						}
+						if (pt_pkt->data_len > 0) {
+							handle_data(pkt, bytes, 0, 0, 0, 0, cur, 0);
+							if (!opts.password) {
```

Comment:
Why checking for opts.password if extendet options used?

Reply:
Extended options take effect after authentication, unless there is no password. If authentication is not required, extended options take effect immediately.